pineappl_py/pineappl/import_only_subgrid.py

```diff
@@ -24,3 +25,24 @@ def __init__(self, array, q2_grid, x1_grid, x2_grid):
         self._raw = PyImportOnlySubgridV1(
             np.array(array), np.array(q2_grid), np.array(x1_grid), np.array(x2_grid)
         )
+
+class ImportOnlySubgridV2(PyWrapper):
+    """
+    Python wrapper object to :class:`~pineappl.pineappl.PyImportOnlySubgridV2`.
+
+    Parameters
+    ----------
+        array : numpy.ndarray(float, dim=3)
+            3-dimensional subgrid content
+        mu2_grid : sequence(float)
```

Comment:
`mu2_grid` can't be a sequence of floats, unless you do some magic later on, e.g.
```py
mu2s = [(mu2, mu2) for mu2 in mu2_grid]
```

But you'd need to account for both the options, so, as a first shot, better to accept a sequence of tuples. In fact, a sequence of `PyMu2` objects.

If you want a simpler constructor, allowing for a sequence of tuples, add a `@classmethod`, such that is always possible to construct from the raw object, in case you already have it.